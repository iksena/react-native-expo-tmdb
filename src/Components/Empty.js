import { Heading, HStack } from 'native-base';

function Empty() {
  return (
    <HStack my={5} justifyContent="center">
      <Heading color="info.500" fontSize="md">
        There is nothing here...
      </Heading>
    </HStack>
  );
}

export default Empty;
