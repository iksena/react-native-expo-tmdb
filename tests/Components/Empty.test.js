import { render } from '../test-utils';
import Empty from '../../src/Components/Empty';

describe('Empty component', () => {
  describe('#render', () => {
    it('should render component', () => {
      const { toJSON } = render(<Empty />);

      expect(toJSON()).toMatchSnapshot();
    });
  });
});
