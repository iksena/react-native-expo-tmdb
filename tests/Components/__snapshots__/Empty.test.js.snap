// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty component #render should render component 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    dataSet={Object {}}
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
        "marginBottom": 20,
        "marginTop": 20,
      }
    }
  >
    <Text
      dataSet={Object {}}
      style={
        Object {
          "backgroundColor": undefined,
          "color": "#0ea5e9",
          "fontFamily": undefined,
          "fontSize": 16,
          "fontStyle": "normal",
          "fontWeight": "700",
          "letterSpacing": 0,
          "lineHeight": 20,
          "textDecorationLine": undefined,
        }
      }
    >
      There is nothing here...
    </Text>
  </View>
</RNCSafeAreaProvider>
`;
